CribblyBackend/Startup.cs

```diff
@@ -67,6 +68,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IMigrati
                 app.UseDeveloperExceptionPage();
             }
             app.UseHttpsRedirection();
+            app.UseSerilogRequestLogging();
```

Comment:
Does this automatically log every request?

Reply:
Yep!